front_end/handlers/ViewPairProgrammingHandler.py

```diff
@@ -0,0 +1,17 @@
+from BaseUserHandler import *
+import datetime as dt
+
+class ViewPairProgrammingHandler(BaseUserHandler):
```

Comment:
This is nitpicky, but please change the name to `ViewPartnerAssignmentsHandler`.